Sources/Controllers/MessagesViewController.swift

```diff
@@ -124,6 +124,13 @@ UICollectionViewDelegateFlowLayout, UICollectionViewDataSource, UIGestureRecogni
         addObservers()
     }
     
+    open override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        if !isFirstLayout {
+            addKeyboardObservers()
```

Comment:
So addObservers doesn't do this?

Reply:
@Kaspik No, addObservers only adds the observer for didReceiveMemoryWarningNotification, it doesn't call addKeyboardObservers

Reply:
So `addKeyboardObservers` wasn't called at all? Okay.. I trust you anyways, so feel free to merge it, but probably to development?